src/Event.h

```diff
@@ -4,7 +4,7 @@
 #include <unordered_map>
 #include <string>
 
-enum class EventType {LOAD_EVENT, DUMMY_EVENT, KEYPRESS_EVENT, EVENT_COUNT};
+enum class EventType {LOAD_EVENT, DUMMY_EVENT, KEYPRESS_EVENT, SETTINGS_EVENT, EVENT_COUNT};
```

Comment:
`SETTINGS_EVENT` isn't a good name because `SETTINGS` isn't an action. `RELOAD_SETTINGS_EVENT` would be better.